static/docs/changelog/0.35.md

```diff
@@ -15,7 +15,7 @@ improvements) we have done in the last few months:
 - 📖 The [Get Started](/doc/get-started/agenda) section has been simplified
   (e.g. to use tags instead of branches) and extended. We have also prepared a
   [Github DVC project ](https://github.com/iterative/example-get-started)that
```

Comment:
minor: add a space after )

Reply:
This was already fixed. But I removed the extra space before `]` in 07fb5b0.